packages/admin-sandbox/admin/pages/homepage.tsx

```diff
@@ -1,7 +1,7 @@
 import { EditPage } from '@contember/admin'
 import { ContentField } from '../components/ContentField'
 
-export const Homepage = (
+export default () => (
```

Comment:
```ts
export default const Homepage = () => {
```

This would keep (AFAIK) the element non-anonymous in the virtual tree for inspection and also would `export default`.

Reply:
That's not syntacticly valid, you need to use `export default function Homepage() { ... }` if you want to keep the element named

Reply:
Right. I didn't check validity. Point is it might be a good idea to keep it named.